src/server/main_service.cc

```diff
@@ -1101,10 +1101,16 @@ void Service::DispatchManyCommands(absl::Span<CmdArgList> args_list,
 
     // MULTI...EXEC commands need to be collected into a single context, so squashing is not
     // possible
-    const bool is_multi =
+    bool is_multi =
```

Comment:
Why not have `is_multi` be `const`? Am I missing a mutation somewhere?

Reply:
I tried to fit it into a single line 😄